File: Dart/controle/switch.dart
import 'dart:math';

main() {
  var nota = Random().nextInt(11);
  print('A nota sorteada foi $nota');

  switch (nota) {
    case 10:
      print('Quadro de Honra!');
      break;
    case 9:
      print('Muito bom!');
      break;
    case 8:
      print('Excelente!');
      break;
    case 7:
      print('Boa!');
      break;
    case 6:
      print('Mais ou menos.');
      break;
    case 5:
      print('Mais sorte da próxima vez.');
      break;
    case 4:
      print('Estude mais!');
      break;
    case 3:
      print('Se esforce mais!');
      break;
    case 2:
      print('Difícil em rapaz!');
      break;
    case 1:
      print('Peça ajuda dos seus colegas!');
      break;
    case 0:
      print('Burro!');
      break;
    default:
      print('Nota inválida!');
  }
}

File: Formação DEV/DEV Express/HTML-CSS-JAVASCRIPT/Controlando-codigo/src/selecao/switch.js
const nota = (Math.random() * 10).toFixed(2);
let conceito = "";

switch (Math.floor(nota)) {
  case 10:
  case 9:
    conceito = "A";
    break;
  case 8:
    conceito = "B";
    break;
  case 7:
  case 6:
    conceito = "C";
    break;
  case 5:
  case 4:
    conceito = "D";
    break;
  case 3:
  case 2:
    conceito = "E";
    break;
  case 1:
  default:
    conceito = "F";
}

console.log(`Nota: ${nota}`);
console.log(`Conceito: ${conceito}`);
